app/controllers/test/components/mentoring/mentor_conversations_lists_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Test::Components::Mentoring::MentorConversationsListsController < ApplicationController
+  def show
+    @mentor = OpenStruct.new(conversations: conversations)
+  end
+
+  private
+  def conversations
+    [
+      {
+        trackIconUrl: "https://assets.exercism.io/tracks/ruby-hex-white.png",
+        menteeAvatarUrl: "https://robohash.org/exercism",
+        menteeHandle: "mentee",
+        exerciseTitle: "Series",
+        isStarred: true,
+        haveMentoredPreviously: true,
+        isNewIteration: true,
+        postsCount: 15,
+        updatedAt: 1.year.ago,
```

Comment:
Convert to `.iso8601`, or pass a format when instantiating `dayJs` instance.

Reply:
I like @SleeplessByte's suggestion about being explicit about the format. What do you think @kntsoriano?

Reply:
Good idea. We should enforce this rule when writing our serializers.